web/packages/teleport/src/AuthConnectors/AuthConnectorTile.tsx

```diff
@@ -0,0 +1,227 @@
+        justifyContent="space-between"
+        alignItems="flex-start"
+        height="100%"
+      >
+        <Icon />
+        <Flex flexDirection="column" alignItems="flex-start" gap={1}>
+          <Flex alignItems="center" gap={2}>
+            <H2>{name}</H2>
+            {isDefault && (
+              <Label
+                kind="secondary"
+                css={`
+                  color: ${props => props.theme.colors.text.slightlyMuted};
+                `}
+              >
+                Default
+              </Label>
+            )}
+          </Flex>
+          <Subtitle2 color="text.slightlyMuted">{customDesc || desc}</Subtitle2>
+        </Flex>
+      </Flex>
+      <Flex
+        flexDirection="column"
+        justifyContent="space-between"
+        alignItems="flex-end"
+        height="100%"
+      >
+        <ActionsSection />
+        {!!onEdit && !!onDelete && (
+          <MenuIcon
+            buttonIconProps={{ style: { borderRadius: `${theme.radii[2]}px` } }}
+          >
+            <MenuItem onClick={onClickEdit}>Edit</MenuItem>
+            <MenuItem onClick={onClickDelete}>Delete</MenuItem>
+          </MenuIcon>
+        )}
+      </Flex>
+    </ConnectorBox>
+  );
+}
+
+/**
+ * LocalConnectorTile is a hardcoded "auth connector" which represents local auth.
+ */
+export function LocalConnectorTile() {
+  return (
+    <AuthConnectorTile
+      key="local-auth-tile"
+      kind="local"
+      id="local"
+      Icon={() => (
+        <Flex
+          alignItems="center"
+          justifyContent="center"
+          css={`
+            background-color: ${props =>
+              props.theme.colors.interactive.tonal.neutral[0]};
+            height: 61px;
+            width: 61px;
+          `}
+          lineHeight={0}
+          p={2}
+          borderRadius={3}
+        >
+          <Password size="extra-large" />
+        </Flex>
+      )}
+      isDefault={true}
+      isPlaceholder={false}
+      isEnabled={true}
+      name="Local Connector"
+      customDesc="Manual auth w/ users local to Teleport"
+    />
+  );
+}
+
+function EnabledIndicator({ isEnabled }: { isEnabled: boolean }) {
+  return (
+    <Flex
+      justifyContent="center"
+      alignItems="center"
+      gap={1}
+      py={1}
+      px={2}
+      css={`
+        background-color: ${isEnabled
+          ? props => props.theme.colors.interactive.tonal.success[0]
+          : props => props.theme.colors.interactive.tonal.neutral[0]};
+        width: 85px;
+        height: 24px;
+        border-radius: 62px;
+      `}
+    >
+      {isEnabled ? (
+        <>
+          <CircleCheck color="interactive.solid.success.default" size="small" />
+          <P3 color="interactive.solid.success.default">Enabled</P3>
+        </>
+      ) : (
+        <>
+          <MinusCircle color="text.disabled" size="small" />
+          <P3 color="text.disabled">Disabled</P3>
+        </>
+      )}
+    </Flex>
+  );
+}
+
+type Props = {
```

Comment:
I'd consider inlining props in the component definition, so that documentation such as JSDocs for props is visible immediately after opening the file, without having to scroll down. In general, I don't think there's much benefit in defining a separate type for props if it's not reused elsewhere.